meteor-app/imports/startup/server/assessment-data.js

```diff
@@ -421,9 +424,28 @@ Meteor.methods({
       return result;
     };
 
-    const assessmentsArray =
-      array_of(n, () => fakeJob())
-        .forEach(r => Assessment.insert(r))
+    // generates all logs up to current status
+    function fakeLogs(id, job) {
+      let logs = []
+      for (let i = 1; i <= job.status; i++) {
+        logs.push({
+          aId: id,
+          user: (i === 1) ? job.assessor : 'Anonymous',
+          status: i,
+          eventType: (i === 1) ? 22 : 20,
```

Comment:
What are these numbers? I thought we had learnt that naked numbers in code are evil :) being unmaintainable as they are. I know this is fake data, but it's just as important that the test code follows the same discipline as the real code.